src/index.ts

```diff
@@ -854,6 +851,16 @@ export class GlobalAuroraRDSSlaveInfra extends cdk.Construct {
 
       cdk.Tags.of(this.dbSubnetGroup).add('Name', 'PublicDBSubnetGroup');
       this.dbSubnetGroup.node.addDependency(rdsVpcSecond);
+    } else {
+      const PrivateSubnet = rdsVpcSecond.selectSubnets({ subnetType: ec2.SubnetType.PRIVATE });
```

Comment:
Can you also write testing in [here](https://github.com/neilkuan/cdk-aurora-globaldatabase/blob/master/test/global-rds.test.ts) ?!
Thanks~

Reply:
You already have tests at `149` and `159` for private/public subnets.